assets/js/modules/subscribe-with-google/components/common/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Subscribe with Google common components.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { default as PrototypeForm } from './PrototypeForm';
```

Comment:
See above, instead of making the whole form a common component, you could have e.g. `PublicationIDInput` and `ProductsInput` as common components here and then use them in a `SetupForm` and `SettingsForm` respectively.